docs/howto/graph-api/links.md

```diff
@@ -0,0 +1,131 @@
+    This means it can only be used to represent `upload_id`, `entry_id`, `dataset_id`, etc., for data that follows a fixed schema (`MongoDB`).
+    It won't work in archives, the corresponding metainfo (definitions), and alike.
+
+## Fuzzy Query
+
+The request configuration allows one to perform fuzzy queries to further filter data before fetching, via the `query` and `pagination` fields.
+
+```py title="request configuration model" hl_lines="4-24"
+class RequestConfig(BaseModel):
+    # ... other fields omitted for brevity ...
+
+    pagination: None | dict = Field(
+        None,
+        description="""
+        The pagination configuration used for MongoDB search.
+        This setting does not propagate to its children.
+        For Token.ENTRIES, Token.UPLOADS and Token.DATASETS, different validation rules apply.
+        Please refer to `DatasetPagination`, `UploadProcDataPagination`, `MetadataPagination` for details.
+        """,
+    )
+    query: None | dict = Field(
+        None,
+        description="""
+        The query configuration used for either mongo or elastic search.
+        This setting does not propagate to its children.
+        It can only be defined at the root levels including Token.ENTRIES, Token.UPLOADS and 'm_datasets'.
+        For Token.ENTRIES, the query is used in elastic search. It must comply with `WithQuery`.
+        For Token.UPLOADS, the query is used in mongo search. It must comply with `UploadProcDataQuery`.
+        For Token.DATASETS, the query is used in mongo search. It must comply with `DatasetQuery`.
+        For Token.GROUPS, the query is used in mongo search. It must comply with `UserGroupQuery`.
+        """,
+    )
+
+    # ... other fields omitted for brevity ...
+```
+
+To instruct the server to perform a query, one need to attach a request config with valid `query` and `pagination` (optional) with the `m_request` key under the **root** level.
+The **root** level is the top-level following the **special tokens**, such as `uploads`, `entries`, etc.
+
+Combined with fuzzy fetching, one can perform filter and fetch.
+For example, if one wants to fetch all entries under an upload with a specific parser name, the request would look like this.
+
+```json title="fuzzy query with pagination" hl_lines="6"
+{
+  "uploads":{
+    "example_upload_id":{
+      "entries":{
+        "m_request":{ "query":{ "parser_name":"desired_parser" } },
+        "*":{
+          "entry_id":{ "m_request":{ "directive":"plain" } },
+          "entry_create_time":{ "m_request":{ "directive":"plain" } }
+        }
+      }
+    }
+  }
+}
+```
+
+As explained in the comments, depending on the specific token, the query must comply with the corresponding query model.
+The `pagination` field shall also comply with the corresponding pagination model.
+Those models are used in conventional REST endpoints, and one can find more details in the corresponding documentation.
```

Comment:
This can be in it's own page, as it deals with scenarios where you do not know the exact resource identifiers. Call the page something like "Queries".